examples/examples_nodejs_test.go

```diff
@@ -196,6 +197,32 @@ func TestSecretsInExplicitProviderNode(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestSSHConnNode(t *testing.T) {
+	token := os.Getenv("DIGITALOCEAN_TOKEN")
+	if token == "" {
+		t.Skipf("Skipping test due to missing DIGITALOCEAN_TOKEN environment variable")
+	}
+	test := getJsOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: path.Join(getCwd(t), "test-ssh-conn", "ts"),
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				ipOutput, ok := stack.Outputs["ipOutput"]
+				assert.True(t, ok)
+				assert.NotEmpty(t, ipOutput)
+				// Due to a bug in the docker client, we add sleep here so our SSH connection does not get refused.
+				time.Sleep(20 * time.Second)
```

Comment:
I'm really confused here. Is the goal to pause between each `pulumi up`? I'm concerned that we are relying on the implementation details of `integration.ProgramTest` here. They could always decide to run `ExtraRuntimeValidation` in another thread.

If this is a known bug, is there an associated issue you can link to?

Reply:
It seems that the actual TS test also has a 20s sleep, so I removed this one.